sdks/python/setup.py

```diff
@@ -193,6 +193,8 @@ def get_version():
     'requests_mock>=1.7,<2.0',
     'tenacity>=5.0.2,<6.0',
     'pytest>=4.4.0,<5.0',
+    # More recent versions of pytest-cov do not support pytest 4.4.0
+    'pytest-cov==2.9.0',
```

Comment:
Alternatively, this could be a dependency only for `py38-cloud`. That makes more sense to me now that I think about it